src/index.less

```diff
@@ -1,443 +1,453 @@
-// // SIDEBAR VARS
-// //
-// @SIDEBAR_MENU_OFFSET_TOP                 : 44px;                // for new topnav integration
-// @SIDEBAR_MENU_Z_INDEX                    : 1020;
-@SIDEBAR_MENU_OPACITY                    : 0.6;
-// @SIDEBAR_MENU_ACTIVE_COLOR               : #ffffff;  //By default
-// @SIDEBAR_MENU_TEXT_ACTIVE_COLOR          : #17273B;
-// @SIDEBAR_MENU_ITEM_HEIGHT                : 36px;
-
-// // COLORS
-// @SIDEBAR_MENU_COLOR                      : #ffffff;
-// @SIDEBAR_MENU_BG_COLOR_LVL1              : #273b54;
-// @SIDEBAR_MENU_BG_COLOR_LVL2              : #3f5167;
-// @SIDEBAR_MENU_BG_COLOR_LVL3              : #59697d;
-// @SIDEBAR_MENU_ACTIVE_ITEM_BG_COLOR       : @SIDEBAR_MENU_ACTIVE_COLOR;
-// @SIDEBAR_MENU_LVL1_ACTIVE_BORDER_COLOR   : @SIDEBAR_MENU_ACTIVE_COLOR;
-// @SIDEBAR_MENU_LVL2_ACTIVE_BORDER_COLOR   : rgba(255, 255, 255, @SIDEBAR_MENU_OPACITY);
-// @SIDEBAR_MENU_VIEW_ALL_ITEM_BORDER       : #8392a4;
-
-// // TOP NAV
-// @SIDEBAR_MENU_MOBILE_TOP_NAV_HEIGHT      : 55px;
-
-// // FONT SIZES
-// @SIDEBAR_MENU_ACTIONS_MENU_FONT_SIZE     : 16px;
-// @SIDEBAR_MENU_LVL1_FONT_SIZE             : 16px;
-@SIDEBAR_MENU_LVL1_ICON_FONT_SIZE        : 20px;
-// @SIDEBAR_MENU_LVL2_FONT_SIZE             : 13px;
-// @SIDEBAR_MENU_LVL2_ICON_FONT_SIZE        : 18px;
-// @SIDEBAR_MENU_LVL3_ICON_FONT_SIZE        : 14px;
-
-// // ACTIONS BUTTON
-@SIDEBAR_MENU_ACTIONS_MENU_BG            : #00a2bf;
-// @SIDEBAR_MENU_ACTIONS_MENU_COLOR         : @SIDEBAR_MENU_COLOR;
-// @SIDEBAR_MENU_ACTIONS_MENU_OPEN_COLOR    : @SIDEBAR_MENU_TEXT_ACTIVE_COLOR;
-
-// /* Hide for mobile, show later */
+@import "../node_modules/ovh-ui-kit/packages/oui-responsive/_variables";
```

Comment:
`~` shorthand couldn't be used? (ref: [rollup-plugin-less-tilde-importer](https://github.com/ovh-ux/rollup-plugin-less-tilde-importer))